ERCS/erc-9999.md

```diff
@@ -0,0 +1,397 @@
+---
+eip: 9999
```

Comment:
```suggestion
eip: 7760
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.